src/plugins/workspace/public/plugin.ts

```diff
@@ -113,14 +115,50 @@ export class WorkspacePlugin implements Plugin<{}, {}, WorkspacePluginSetupDeps>
     this.managementCurrentWorkspaceIdSubscription = currentWorkspaceId$.subscribe(
       (currentWorkspaceId) => {
         if (currentWorkspaceId) {
-          ['settings', 'dataSources'].forEach((appId) =>
+          ['settings', 'dataSourceManagement'].forEach((appId) =>
             management.sections.section.opensearchDashboards.getApp(appId)?.disable()
           );
         }
       }
     );
   }
 
+  /**
+   * Add workspace overview page to breadCrumb
+   * @param core CoreStart
+   * @private
+   */
+  private addWorkspaceToBreadCrumb(core: CoreStart) {
+    this.chromeSubscription = combineLatest([
+      core.workspaces.currentWorkspace$,
+      core.chrome.getBreadcrumbs$(),
+    ]).subscribe(([currentWorkspace, breadCrumbs]) => {
+      if (currentWorkspace && breadCrumbs && breadCrumbs.length > 0) {
+        const workspaceNotInBreadcrumbs =
+          breadCrumbs.findIndex((breadcrumb: ChromeBreadcrumb) => {
+            return breadcrumb.text === currentWorkspace.name;
+          }) === -1;
+        if (workspaceNotInBreadcrumbs) {
+          const workspaceCrumb: ChromeBreadcrumb = {
+            text: currentWorkspace.name,
+            onClick: () => {
+              core.application.navigateToApp(WORKSPACE_OVERVIEW_APP_ID);
+            },
+          };
+          const homeCrumb: ChromeBreadcrumb = {
+            text: 'Home',
+            onClick: () => {
+              core.application.navigateToApp('home');
+            },
+          };
+          breadCrumbs.splice(0, 0, homeCrumb, workspaceCrumb);
```

Comment:
Should you use `slice` instead of `splice`? `splice` will change the array in place, I guess that's why the subscribe to `breadcrumb$` not refreshing the breadcrumb, because it returns the same array instance.

Reply:
Changed to `Promise.resolve().then(() => core.chrome.setBreadcrumbs(breadCrumbs));` as workaround

Reply:
Just curious why using promise could solve the issue? :D

Reply:
> Just curious why using promise could solve the issue? :D

Yeah, I am looking into it, it does not make sense to me.

Reply:
might be local build cache issue, after clean all build caches and start with a refresh package, it works well.